include/rcutils/error_handling.h

```diff
@@ -314,6 +314,93 @@ RCUTILS_PUBLIC
 void
 rcutils_reset_error(void);
 
+// Helper macro for nested macro expansion
+#define _RCUTILS_EXPAND(x) x
+
+/// Set the error message, as well as append the current file, line number, and previous error.
```

Comment:
I would just say the previous error. And then mention the get error function is used. Otherwise you're repeating its documentation here. And if we ever change what it includes this will then be out of date.